src/BaseComponent.jsx

```diff
@@ -108,7 +108,8 @@ export default class Component extends React.PureComponent {
         else if (!this[_scope]) {
             // Note: We don't error here, because sometimes people decorate a class that's not a component with @Component.
             // If scope is actually used, there will still be an error further down the line, but this warning should help explain it!
-            console.warn('A top-level component was instantiated without a forked scope - please change to @Component({ fork: true })');
+            let className = this.constructor.name;
+            console.warn(`\`${className}\` was instantiated at the top-level without a forked scope - please change to @Component({ fork: true })`);
```

Comment:
I still wish there was a way we could detect/correct this without them having to specify `{ fork: true }`. I know you did the research and I did too (that `_owner` used to be a thing in React but isn't supported)... it's just unfortunate that end-users learning Twist need to understand what "fork" means (or assume that it's magic) in order to render a basic component.

Reply:
I opened #7 to fix this separately.